parity/cli.rs

```diff
@@ -219,6 +219,8 @@ Footprint Options:
                            fast - maintain journal overlay. Fast but 50MB used.
                            auto - use the method most recently synced or
                            default to fast if none synced [default: auto].
+  --pruning-history NUM    Set a number of recent states to keep when pruning
+                           is active. [default: 64].
```

Comment:
Isn't ~128 a bit safer? Although the mainnet isn't often susceptible to reorgs beyond 5 or so, the testnet, ETC, expanse, or private chains don't have quite the same level of stability.